addons/cookoff/functions/fnc_cookOffBox.sqf

```diff
@@ -44,9 +44,9 @@ if (local _vehicle) then {
 
         // These functions are smart and do all the cooking off work
         if (local _box) then {
-            _box call FUNC(secondaryExplosions);
             if (_box getVariable [QGVAR(enableAmmoCookoff), GVAR(enableAmmoCookoff)]) then {
```

Comment:
@SilentSpike - I merged as is, we can look at settings in another pr
Should also look at incerate grenades and make sure it's using the right ace_setting?
https://github.com/acemod/ACE3/blob/master/addons/grenades/functions/fnc_incendiary.sqf#L172